Page de connexion

valerianchar opened the pull request:
![image](https://github.com/user-attachments/assets/047b2910-cb54-4c2f-ab3f-5a66fe2e127e)


![image](https://github.com/user-attachments/assets/ffb00e7b-3fbd-4564-84d2-76baa781f284)

Commit:
Page de connexion sans connexion

Peyronon-Arno commented on src/app/views/connexion/connexion.component.html:
```diff
@@ -0,0 +1,40 @@
+                class="connexion__div__form__content__input"
+                [formControl]="form.controls.password"
+                [errorMessages]="{
+                    required: 'Mot de passe requis',
```
pas besoin de required

Peyronon-Arno commented on src/app/views/connexion/connexion.component.html:
```diff
@@ -0,0 +1,40 @@
+
+    <div class="connexion__div__links">
+        <p class="connexion__div__links__link" (click)="newAccount()">
+          Nouveau ? 
```
Créer un compte ou un truc du genre

Commit:
Update login form messages and text

- Removed the required password error message.
- Changed "Nouveau ?" to "Pas de compte ?" for clarity.
